bench-tps/src/cli.rs

```diff
@@ -290,10 +290,10 @@ pub fn build_args<'a, 'b>(version: &'b str) -> App<'a, 'b> {
                 .help("Submit transactions with a TpuClient")
         )
         .arg(
-            Arg::with_name("tpu_use_quic")
-                .long("tpu-use-quic")
+            Arg::with_name("tpu_disable_quic")
+                .long("tpu-disable-quic")
```

Comment:
I was wondering if we needed to not break the bench-tps api by removing `--tpu-use-quic`, but I guess v1.11.5 already did this breaking...